packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -79,11 +79,11 @@ class ReorderableListView extends StatefulWidget {
 
   /// Called when a list child is dropped into a new position to shuffle the
   /// underlying list.
-  /// 
+  ///
   /// This [ReorderableListView] calls [onReorder] after a list child is dropped
   /// into a new position.
   final OnReorderCallback onReorder;
-  
+
   @override
   State<StatefulWidget> createState() {
```

Comment:
I'd mentioned this before. CreateState methods like this are boilerplate and the convention for their format is:
```
_ReorderableListViewState createState() => new _ReorderableListViewState();
```

Reply:
Done.